client/src/test/java/io/openlineage/client/OpenLineageTest.java

```diff
@@ -0,0 +1,44 @@
+package io.openlineage.client;
+
+import java.util.Arrays;
+import java.util.List;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+
+import io.openlineage.client.OpenLineage.Dataset;
+import io.openlineage.client.OpenLineage.Job;
+import io.openlineage.client.OpenLineage.JobFacets;
+import io.openlineage.client.OpenLineage.Run;
+import io.openlineage.client.OpenLineage.RunFacets;
+import io.openlineage.client.OpenLineage.RunStateUpdate;
+
+public class OpenLineageTest {
+
+  @Test
+  public void jsonSerialization() throws JsonProcessingException {
```

Comment:
_Minor_: I wondering if we want to load the expected json from a file, then check that against the `OpenLineage.RunStateUpdate` object? or maybe define the json string in-line:

```
String expectedJson = "{...";
```

Reply:
agreed, this needs to be improved.